client/src/main/scala/com.gu.contentapi.client/ContentApiClient.scala

```diff
@@ -82,15 +84,12 @@ trait ContentApiClient {
     * @param f a result-processing function
```

Comment:
Could we update the comments in line with the changes? I think the `Q` was removed in one block, but there's no mention of `E` or `M`. Unless we can give those more meaningful identifiers and not need the comments. This applies in several places below.

Reply:
Thanks Justin! I've added/updated scaladoc for those type parameters with commit a789575.